cardano-db/src/Cardano/Db/Schema.hs

```diff
@@ -439,14 +439,14 @@ share
 
   -- A table containing a list of delisted pools.
   DelistedPool
-    poolId              Types.PoolId        sqltype=text
+    poolId              Types.DbPoolId        sqltype=text
```

Comment:
I am still not sure what type that field should be. Maybe it should even be an index into the `pool_update` table.